mmv1/products/certificatemanager/api.yaml

```diff
@@ -212,4 +212,207 @@ objects:
+        name: 'labels'
+        description: |
+          Set of labels associated with a Certificate Map resource.
+      - !ruby/object:Api::Type::Array
+        name: 'gclbTargets'
+        description: |
+          A list of target proxies that use this Certificate Map
+        output: true
+        item_type: !ruby/object:Api::Type::NestedObject
+          properties:
+            - !ruby/object:Api::Type::Array
+              name: 'ipConfigs'
+              description: |
+                An IP configuration where this Certificate Map is serving
+              item_type: !ruby/object:Api::Type::NestedObject
+                properties:
+                  - !ruby/object:Api::Type::String
+                    name: 'ipAddress'
+                    description : |
+                      An external IP address
+                  - !ruby/object:Api::Type::Array
+                    name: 'ports'
+                    description : |
+                      A list of ports
+                    item_type: Api::Type::Integer
+            - !ruby/object:Api::Type::String
+              name: 'targetHttpsProxy'
+              description: |
+                Proxy name must be in the format projects/*/locations/*/targetHttpsProxies/*.
+                This field is part of a union field `target_proxy`: Only one of `targetHttpsProxy` or
+                `targetSslProxy` may be set.
+            - !ruby/object:Api::Type::String
+              name: 'targetSslProxy'
+              description: |
+                Proxy name must be in the format projects/*/locations/*/targetSslProxies/*.
+                This field is part of a union field `target_proxy`: Only one of `targetHttpsProxy` or
+                `targetSslProxy` may be set.
+  - !ruby/object:Api::Resource
+      name: 'CertificateMapEntry'
+      base_url: 'projects/{{project}}/locations/global/certificateMaps'
+      create_url: 'projects/{{project}}/locations/global/certificateMaps?certificateMapId={{name}}'
+      self_link: 'projects/{{project}}/locations/global/certificateMaps/{{name}}'
+      update_verb: :PATCH
+      update_mask: true
+      description: |
+        CertificateMapEntry is a list of certificate configurations,
+        that have been issued for a particular hostname 
+      async: !ruby/object:Api::OpAsync
+        operation: !ruby/object:Api::OpAsync::Operation
+          path: 'name'
+          base_url: '{{op_id}}'
+          wait_ms: 1000
+        result: !ruby/object:Api::OpAsync::Result
+          path: 'response'
+        status: !ruby/object:Api::OpAsync::Status
+          path: 'done'
+          complete: true
+          allowed:
+            - true
+            - false
+        error: !ruby/object:Api::OpAsync::Error
+          path: 'error'
+          message: 'message'
+      parameters:
+        - !ruby/object:Api::Type::String
+          name: 'name'
+          required: true
+          input: true
+          url_param_only: true
```

Comment:
I don't think that this will be a url param? We'll have to test it out, but it looks like it will be in the request body.